recipe/fix-osx.patch

```diff
@@ -106,7 +106,7 @@ index 046e367..645f15a 100644
  
  /* accounting object for the main memory */
 -static pocl_global_mem_t system_memory;
-+static pocl_global_mem_t system_memory = POCL_LOCK_INITIALIZER;
++static pocl_global_mem_t system_memory = {POCL_LOCK_INITIALIZER, 0, 0, 0};
```

Comment:
This is what was suggested upstream. It's more portable, but shouldn't matter in linux and osx

Reply:
Could you add a link to the location of the suggestion?